addons/main/script_macros.hpp

```diff
@@ -97,5 +97,9 @@
 // Time functions for accuracy per frame
 #define ACE_tickTime (ACE_time + (diag_tickTime - ACE_diagTime))
 
+#define ACE_LOG(module,level,message) diag_log text format [QUOTE([ACE] (%1) %2: %3), module, level, message];
+#define ACE_LOGERROR(message) ACE_LOG(QUOTE(COMPONENT),"ERROR",message)

```

Comment:
That won't work. Just tested it and it would result in it being `#common` instead of `"common"` for example.
I assume it's because it's inside another macro call? Not sure on that though.